__tests__/__snapshots__/ordCdsrc.test.js.snap

```diff
@@ -0,0 +1,208 @@
+          "accessStrategies": [
+            {
+              "type": "open",
+            },
+          ],
+          "mediaType": "application/xml",
+          "type": "edmx",
+          "url": "/.well-known/open-resource-discovery/v1/api-metadata/CatalogService.edmx",
+        },
+      ],
+      "shortDescription": "CatalogService",
+      "title": "CatalogService",
+      "version": "1.0.0",
+      "visibility": "public",
+    },
+  ],
+  "description": "this is my custom description",
+  "eventResources": [
+    {
+      "description": "CAP Event resource describing events / messages.",
+      "extensible": {
+        "supported": "no",
+      },
+      "ordId": "sap.test.cdsrc.sample:eventResource:AdminService:v1",
+      "partOfGroups": [
+        "sap.cds:service:sap.test.cdsrc.sample:AdminService",
+      ],
+      "partOfPackage": "sap.test.cdsrc.sample:package:capirebookshopordsample-event:v1",
+      "releaseStatus": "beta",
+      "resourceDefinitions": [
+        {
+          "accessStrategies": [
+            {
+              "type": "open",
+            },
+          ],
+          "mediaType": "application/json",
+          "type": "asyncapi-v2",
+          "url": "/.well-known/open-resource-discovery/v1/api-metadata/AdminService.asyncapi2.json",
+        },
+      ],
+      "shortDescription": "Example ODM Event",
+      "title": "ODM capirebookshopordsample Events",
+      "version": "1.0.0",
+      "visibility": "public",
+    },
+    {
+      "description": "CAP Event resource describing events / messages.",
+      "extensible": {
+        "supported": "no",
+      },
+      "ordId": "sap.test.cdsrc.sample:eventResource:CatalogService:v1",
+      "partOfGroups": [
+        "sap.cds:service:sap.test.cdsrc.sample:CatalogService",
+      ],
+      "partOfPackage": "sap.test.cdsrc.sample:package:capirebookshopordsample-event:v1",
+      "releaseStatus": "beta",
+      "resourceDefinitions": [
+        {
+          "accessStrategies": [
+            {
+              "type": "open",
+            },
+          ],
+          "mediaType": "application/json",
+          "type": "asyncapi-v2",
+          "url": "/.well-known/open-resource-discovery/v1/api-metadata/CatalogService.asyncapi2.json",
+        },
+      ],
+      "shortDescription": "Example ODM Event",
+      "title": "ODM capirebookshopordsample Events",
+      "version": "1.0.0",
+      "visibility": "public",
+    },
+  ],
+  "groups": [
+    {
+      "groupId": "sap.cds:service:sap.test.cdsrc.sample:AdminService",
+      "groupTypeId": "sap.cds:service",
+      "title": "Admin Service",
+    },
+    {
+      "groupId": "sap.cds:service:sap.test.cdsrc.sample:CatalogService",
+      "groupTypeId": "sap.cds:service",
+      "title": "Catalog Service",
+    },
+  ],
+  "openResourceDiscovery": "1.9",
+  "packages": [
+    {
+      "description": "Description for capire bookshop ord sample",
+      "ordId": "sap.test.cdsrc.sample:package:capirebookshopordsample-api:v1",
```

Comment:
it looks it could generate multi packages by appending different suffix, I guess it's valid @Fannon.

Reply:
Ok, so does it now append suffixes or not? I thought for now, it just generates one package and assigns all the ORD resources to it? But in the future, we would add the option to automatically add the suffixes and assign the resources to the right kind of package :)

Reply:
Yes, the [original code](https://github.com/cap-js/ord/blob/main/lib/templates.js#L349) has the logic to append suffix to different type.